### PR TITLE
Master

### PR DESCRIPTION
**Pull request by svenbeyer13**
Hi Mark, 
I am fairly new to GitHub, so I hope I am doing this "committing" and "pulling" correctly.
...

### CODE REVIEW

**Comment by mlenser** [comment]
@Rouby can you review the alterations that @svenbeyer13 put forward?

I'd suggest referencing the German translation of skill from earlier ediitons. I think 3.5 was translated to German - maybe 4e as well.

**Comment by svenbeyer13** [comment]
I am happy to also revise the rest of the german translations, making them easier to read/understand for German native speakers.

File: 5eShaped/precompiled/translations/de.json [hunk 1]
```diff
@@ -55,17 +55,17 @@
     "athletics": "Athletik",
     "deception": "Täuschung",
     "history": "Geschichte",
-    "insight": "Einsicht",
-    "intimidation": "Einschüchterung",
-    "investigation": "Investigation",
+    "insight": "Motiv erkennen",
```
Comment (Rouby) [inline 5eShaped/precompiled/translations/de.json]: I would like to stick to single-word skills where possible.

"Einsicht" was used in other german 5e sheets. I think 4e used "Einschätzen".

**Comment by mlenser** [comment]
I didn't make them. @Rouby, who is also native, did. I would suggest you guys collaborate - and primarily look at old edition translations and use those if they are good.

File: 5eShaped/precompiled/translations/de.json [hunk 2]
```diff
@@ -55,17 +55,17 @@
...
+    "intimidation": "Einschüchtern",
```
Comment (Rouby) [inline 5eShaped/precompiled/translations/de.json:59]: Why change this to the verb form and change "Entdecken" to "Wahrnehmung"?

File: 5eShaped/precompiled/translations/de.json [hunk 3]
```diff
@@ -55,17 +55,17 @@
...
+    "investigation": "Untersuchung",
     "medicine": "Medizin",
```
Comment (Rouby) [inline 5eShaped/precompiled/translations/de.json:61]: Maybe change "Medizin" to "Heilkunde"

**Comment by Rouby** [comment]
I added some comments. They are just my input on those and we can discuss it if you like, rest seems fine / better than what I initially submitted :)

File: 5eShaped/precompiled/translations/de.json (hunk 1 above)
Comment (svenbeyer13) [inline 5eShaped/precompiled/translations/de.json]: the "insight" term changed in its content over the versions of DnD. I never played 4th edition, but the way the skill is described, it means something like "sense motive". 
Both words "Einsicht" and "Einschätzen" are less ideal, where I would from a language perspective prefer "Einschätzen" over "Einsicht".

File: 5eShaped/precompiled/translations/de.json (hunk 3 above)
Comment (svenbeyer13) [inline 5eShaped/precompiled/translations/de.json:61]: good point. "Heilkunde" was the Term Common in 3.5

File: 5eShaped/precompiled/translations/de.json (hunk 1 above)
Comment (mlenser) [inline 5eShaped/precompiled/translations/de.json]: Insight is definitely the intention in 4/5e which differs a bit from sense motive.

File: 5eShaped/precompiled/translations/de.json (hunk 2 above)
Comment (svenbeyer13) [inline 5eShaped/precompiled/translations/de.json:59]: "Entdecken" equals "detection", where "Wahrnehmung" equals "perception"
"Einschüchtern" is the term used in 3.5. I did not think about whehter or nor we are looking at nouns versus verbs, but if you feel this is important, please keep "Einschüchterung".

**Comment by svenbeyer13** [comment]
OK, revised the commit. What's next?

File: 5eShaped/precompiled/translations/de.json (hunk 2 above)
Comment (Rouby) [inline 5eShaped/precompiled/translations/de.json:59]: Neverwinter Nights (based on 3.5) used "Entdecken", thats why I chose it too. Both seem fine I guess.

**Comment by mlenser** [comment]
@Rouby are you happy with his final changes?

**Comment by Rouby** [comment]
Yep

**Pull request merged**